### ISSUE TITLE
Deck Reports Shaders Still Hogging Internal Drive Storage

### ISSUE THREAD

**Issue by InklingGirl** [CryoByte33/steam-deck-utilities#162]
Despite all being synced to the microSD. Deck is convinced shaders are taking up 15 GB of the internal drive & that I only have mere MBs of storage left, is this something I should worry about? Thanks.

**Comment by CryoByte33** [comment]
I'm not sure how the Steam client calculates the usage, but it's possible that it's seeing the size "through" the symlink. If you can verify that the shaders themselves are on the MicroSD then you should be good!

As a note, only games that are installed on the MicroSD have their shaders moved there, and you may also want to check that there are no shaders for uninstalled games with the clean functionality in CU.

**Comment by InklingGirl** [comment]
Yes, everything is installed on the microSD & been run, so it's all definitely on the microSD & merely symlinked. I should also say, it's not just the Steam client reporting a lack of space (most of which it attributes to shaders), but also KDE Plasma's `dolphin` also saying my home directory is full.

**Comment by CryoByte33** [comment]
That's odd, have you tried filelight to see where that space is being used on a directory basis? It's possible that /home is full for some other reason and this looks suspect 🤔
